src/utils.js

```diff
@@ -15,6 +15,9 @@ export const getBinaryFileURL = ({ version, path }) =>
 export const removeAppPathPrefix = (path, appName) =>
   path.replace(new RegExp(`${appName || 'RnDiffApp'}/`), '')
 
+export const getOriginalPath = (path, appName) =>
+  path.replace(new RegExp(`${appName}`), 'RnDiffApp')
```

Comment:
@pvinis Following this change, for some reason, ios paths show the app name twice at the beginning resulting in a 404 again.

eg https://github.com/react-native-community/rn-diff-purge/raw/release/0.59.1/RnDiffAppRnDiffApp/package.json

notice `RnDiffAppRnDiffApp` in the path just after the version number.

I'm wondering it it has something to do with my implementation of the `replace` method.

Should it maybe just be this...?

```suggestion
  path.replace(appName, 'RnDiffApp')
```

Reply:
I tried this, it's not that... I'll try to fix this and open a new pull request, if you have any idea, I'd welcome any pointers ;) Thanks.

Reply:
in fact, this seems to be a problem with files that have the app name in it (like some of the ios ones)...

Reply:
Should be fixed in https://github.com/react-native-community/upgrade-helper/pull/209